src/kiwixapp.cpp

```diff
@@ -346,25 +346,25 @@ void KiwixApp::setMonitorDir(const QString &dir) {
 
 void KiwixApp::createAction()
 {
-    CREATE_ACTION_ICON_SHORTCUT(KiwixServeAction, "share", gt("local-kiwix-server"), QKeySequence(Qt::CTRL, Qt::Key_I));
+    CREATE_ACTION_ICON_SHORTCUT(KiwixServeAction, "share", gt("local-kiwix-server"), QKeySequence(Qt::CTRL | Qt::Key_I));
```

Comment:
I think that this entire commit better be merged with the earlier one that replaced `+` with `,`.